pkg/controller/storage/account/account.go

```diff
@@ -117,21 +110,6 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		return reconcile.Result{}, err
 	}
 
-	if !resource.IsConditionTrue(b.GetCondition(runtimev1alpha1.TypeReferencesResolved)) {
```

Comment:
Were we previously resolving references here even though the storage types do not have any ref fields?

Reply:
Yeah, exactly. I figured it made more sense to remove this logic than to port it to the new resolver.